boards/arm/stm32wl5/nucleo-wl55jc/src/stm32_leds.c

```diff
@@ -188,7 +215,13 @@ void board_autoled_off(int state)
 
 uint32_t board_userled_initialize(void)
 {
-  /* Already initialized by stm32_led_initialize. */
+  /* Leds are already initialized by stm32_led_initialize. */
+
+#ifdef CONFIG_ARCH_BOARD_NUCLEO_WL55JC_DEMO_LED_IRQ
+  /* Configure B3 button to fire an interrupt on falling edge (on press)  */
+
+  stm32wl5_gpiosetevent(GPIO_BUTTON3, 0, 1, 0, button3_led, NULL);
```

Comment:
Let's change here as well
```suggestion
  stm32wl5_gpiosetevent(GPIO_BUTTON3, false, true, false, button3_led, NULL);
```